infra/frontend/app-config/env-config/environment_variables.tf

```diff
@@ -83,6 +83,10 @@ locals {
       manage_method     = "manual"
       secret_store_name = "/${var.app_name}/${var.environment}/feature-saved-searches-on"
     },
+    FEATURE_APPLY_FORM_PROTOTYPE_OFF = {
+      manage_method     = "manual"
+      secret_store_name = "/${var.app_name}/${var.environment}/feature-saved-searches-on"
```

Comment:
not sure what's going on here - do we actually want to have the apply forms off whenever saved search is on, or is this a typo and should point to its own variable in SSM?

Also, could we make this a positive polarity to match the rest of the recent feature flag env vars?